src/quick-lint-js/parse.h

```diff
@@ -2481,9 +2484,9 @@ class parser {
     bool parsed_body =
         this->parse_and_visit_statement_disallowing_declaration(v);
     if (!parsed_body) {
+      const char8 *here = this->lexer_.end_of_previous_token();
       this->error_reporter_->report(error_missing_body_for_for_statement{
```

Comment:
Perhaps we should include a *NOTE* mentioning where the `for` loop starts. See E141 for example.